.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "cmake/deps/emsdk"]
```

Comment:
I intentionally avoid the git submodule to decrease this project complexity.
Can we pass the emsdk DIR in the build command argument?

Reply:
Sure, we can download emsdk in the pipeline. BTW, I am curious why git submodule will increase complexity.